main.go

```diff
@@ -315,7 +317,21 @@ func main() {
 			log.Printf("%v\n", err)
 			os.Exit(64)
 		}
-		exitCode := RunWorker()
+		monitor = NewMonitor(
+			config.Project,
+			tcauth.New(&tcclient.Credentials{
+				ClientID:    config.ClientID,
+				AccessToken: config.AccessToken,
+				Certificate: config.Certificate,
+			}),
+			config.LogLevel,
+			nil,
+		)
+
+		exitCode := INTERNAL_ERROR
+		incidentID := monitor.CapturePanic(func() {
+			exitCode = RunWorker()
+		})
```

Comment:
> If a monitor instance isn't created during tests, is this handled gracefully?
Do you mean monitoring calls?

>  The RunWorker method is designed not to panic, so I don't like it being able to panic now, which is a change in behavior that could affect test results.
Hrm, if a test expects `RunWorker` to panic, shouldn't it capture it? `Panic` is something orthogonal and unexpected by nature, so caller either shouldn't expect any function to panic or expect any function to...